src/NServiceBus.AcceptanceTests/Core/SubscriptionMigration/AcceptanceTestingTransportServer.cs

```diff
@@ -39,8 +35,9 @@ public async Task<EndpointConfiguration> GetConfiguration(RunDescriptor runDescr
             await persistenceConfiguration.Configure(endpointConfiguration.EndpointName, configuration, runDescriptor.Settings, endpointConfiguration.PublisherMetadata);
             runDescriptor.OnTestCompleted(_ => persistenceConfiguration.Cleanup());
 
-            configurationBuilderCustomization(configuration);
+            await configurationBuilderCustomization(configuration);
```

Comment:
```suggestion
            await configurationBuilderCustomization(configuration).ConfigureAwait(false);
```

Reply:
Or consistently remove it if we generally don add it in the ATTs. I don know what the latest state is there

Reply:
it seems ConfigureAwait is generally not used, so I removed it